.github/workflows/main-push.yaml

```diff
@@ -27,17 +32,20 @@ jobs:
         uses: helm/chart-testing-action@v2.4.0
 
       - name: Run chart-testing (lint)
-        run: ct lint
+        run: ct lint --charts ${{ matrix.chart-name }}
```

Comment:
This will pass for `ingress` given that it doesn't have any ct related manifests?

Reply:
It does. `ct lint` and `ct install` are different tasks. `ct install` won't pass which is why it's got an `if:` block